libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -348,35 +346,23 @@ - (void)dealloc
 - (BOOL)loginWithCompletion:(SFOAuthFlowSuccessCallbackBlock)completionBlock
                     failure:(SFOAuthFlowFailureCallbackBlock)failureBlock
 {
-    return [self loginWithCompletion:completionBlock failure:failureBlock account:nil];
+    return [self loginWithCompletion:completionBlock failure:failureBlock credentials:self.oauthCredentials];
```

Comment:
Minor: `oauthCredentials` isn't defined as a property—and probably rightly so, given its function.  We should use method accessor notation instead, e.g. `[self oauthCredentials]`.